contributing.md

```diff
@@ -0,0 +1,18 @@
+# How to contribute to Scoreboard
+
+## Steps to contributing
+
+- Fork the repository
+- Create a branch for your changes
+- Make the changes you'd like to contribute. See the "types of contributions" list above to learn more about what we're looking for
+- Submit a pull request
+
+Before changes can be merged into master, they _must_ be accompanied by an update to the [CHANGELOG.md](https://github.com/developmentseed/scoreboard/blob/develop/CHANGELOG.md). Please file all new changes under the "Unreleased" heading in CHANGELOG.md, under the appropriate subheading of "Added," "Changed" or "Fixed."
```

Comment:
Let's add here that we use https://keepachangelog.com/en/1.0.0/

Reply:
@kamicut That's included in the Changelog itself... I don't think it's vital to link out from this page as well